util/afero.go

```diff
@@ -100,10 +101,7 @@ func CopyFs(srcFs, dstFs afero.Fs, src, dst string, force bool) error {
 		}
 		//perm := info.Perm()
 		perm := info.Mode() // TODO: is this correct?
-		p := name
```

Comment:
Hmmm.. Is it really that simple? If so, sweet. I'll try and test and merge this soon. Thanks!

Reply:
Works for me. :)